src/main/java/io/antmedia/rest/BroadcastRestService.java

```diff
@@ -268,7 +271,22 @@ public List<Broadcast> getBroadcastList(@ApiParam(value = "This is the offset of
 			@ApiParam(value = "\"asc\" for Ascending, \"desc\" Descending order", required = false) @QueryParam("order_by") String orderBy,
 			@ApiParam(value = "Search parameter, returns specific items that contains search string", required = false) @QueryParam("search") String search
 			) {
-		return getDataStore().getBroadcastList(offset, size, typeBy, sortBy, orderBy, search);
+		List<Broadcast> broadcastList = getDataStore().getBroadcastList(offset, size, typeBy, sortBy, orderBy, search);
+		boolean isCluster = Objects.requireNonNull(getAppContext()).containsBean(IClusterNotifier.BEAN_NAME);
+		if (isCluster) {
```

Comment:
Thank you for the solution. 

On the other hand, I think this is not the correct place to fix this issue because getBroadcastList job is to get the broadcast list. Its job is not recovery of the status of the streams in the cluster. Even if this is a solution, these kind of fixes probably will make the code hard to maintain and secondly if this method is not called(very likely to happen) then this fix will not work. So I'm not accepting this fix. 

The correct place is probably where the problem occurs. I'll send a fix for this PR because some users are waiting for this issue. 

Gentle reminder from Manifesto
>Solve problems permanently. Don’t waste time addressing the same issue multiple times.

Reply:
I appreciate your review, and I believe your point is entirely valid. During my investigation of the bug, I encountered a challenge in updating the stream in the event of a crash occurring in the origin node. Given that this is a crash scenario, executing code at the moment of occurrence is not feasible.
I integrated this code into the getBroadcastList function because, in cases where a node becomes inaccessible and a stream is lost, users will inevitably need to use this API endpoint to verify statuses. By incorporating this code here, the list presented to users will accurately reflect the statuses. Additionally, if a broadcast isn't live, it will be successfully deleted through a delete call.  We have the option to relocate the solution to an alternative function or location, or even trigger a similar block at regular intervals. I'm genuinely interested in your proposed solution and eager to explore it!

Reply:
Thank you @lastpeony 

I've sent the PR below. You can check it and we can discuss it. I don't claim that my solution is the best way to fix the problem. I can say that this is the thing I can do at this stage.  So please feel free to share any of your thoughts. Then we, together, find the better way in approaching these kind of cases.